ChelseasFieldJournal/src/GUI/GuiDialogToolJournal.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Cairo;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Vintagestory.API.Client;
+using Vintagestory.API.Common;
+using Vintagestory.API.Config;
+using Vintagestory.API.MathTools;
+
+namespace ChelseasFieldJournal
+{
+    public class GuiDialogToolJournal : GuiDialogGeneric, IDisposable
+    {
+        public InventoryBase Inventory { get; set; }
+        
+        private int leftPageNumber { get; set; }
+        private int rightPageNumber { get; set; }
+        private Item ItemBook { get; set; }
+
+        public GuiDialogToolJournal(string DialogTitle, EntityPlayer playerIn, ICoreClientAPI capi) : base(DialogTitle, capi)
+        {
+            this.SetupDialogJournal();
+            this.RunAllSetupDialogs();
+            this.lastRedrawMs = this.capi.ElapsedMilliseconds;
+        }
+
+        private void RunAllSetupDialogs()
+        {
+            //this.SetupDialogDiscoveredEntries();
+            
+        }
+
+        private void SetupDialogJournal()
+        {
+            ElementBounds dialogBounds = ElementStdBounds.AutosizedMainDialog.WithAlignment(EnumDialogArea.CenterMiddle);//.WithFixedAlignmentOffset(0.0, 20.0);
+            ElementBounds JournalBlankBounds = ElementBounds.Fixed(0.0, 0.0, 1091, 700);
+            ElementBounds TestIconBounds = ElementBounds.Fixed(650.0, 100.0, 256.0, 196.0);
+            ElementBounds ItemStackIconBounds = ElementBounds.Fixed(200.0, 200.0, 256.0, 256.0);
+            ElementBounds childBounds = new ElementBounds().WithSizing(ElementSizing.FitToChildren);
+            IGuiAPI gui = this.capi.Gui;
+            string str = "gui-dialog-journal";
+            AssetLocation test = new AssetLocation("fieldjournal", "textures/dialogs/test.png");
```

Comment:
Move this to the top of the method under `string dialogueId = "gui-dialog-journal"`